kedro-telemetry/RELEASE.md

```diff
@@ -1,4 +1,6 @@
-# Upcoming release 0.4.0
+# Upcoming release
+
+# Release 0.4.0
 * Updated the plugin to generate an unique UUID for each user of `kedro-telemetry`.
```

Comment:
I feel like we need to add more docs, we should also update the docs (maybe in README.md too). Do we now have `.telemetry` and `telemetry.toml` at the same time?

Reply:
I agree. It appears we need to revise the Kedro documentation to reflect the updated method for identifying user identity with storing it in the `telemetry.toml` file. I don't think that we should change `README.md` to save consistency.

`Hashed Username: An anonymized representation of the user’s computer username.`

Reply:
xref https://github.com/kedro-org/kedro/pull/3805